TAMULib/vufind
# Further Underlining Rule Revisions

# Description

This change further revises the CSS rules for underlining. Book cover images on the search results and record pages are no longer underlined under any circumstance, but the previous underlining rules for text links are retained (so that links have dotted lines which change to solid lines when the user hovers over them).

## Type of change


# Testing Procedures

This change is live on [catalog-pre](https://catalog-pre.library.tamu.edu/). Run a search, and click on any of the results. There should not be any underlining on the cover images on either the search results or record pages. Text links should have dotted underlines, which change to solid lines when the user hovers over them.
